main.go

```diff
@@ -26,7 +26,10 @@ func main() {
 
 	api := slack.New(cnf.SlackToken)
 
-	c, err := selectChannel(coll, api)
+	var ignore Channels
+	ignore = append(ignore, Channel{Name: cnf.DestChan})
```

Comment:
Let's get this on one line by using a literal

```
ignore := Channels{Channel{Name: cnf.DestChan}}
```

You could go crazy and leave off the field name like

```
ignore := Channels{Channel{cnf.DestChan}}
```

And when you have a slice of a thing you don't have to specify the thing type every time so you could leave off `Channel` too

```
ignore := Channels{{Name: cnf.DestChan}}
```

Or go super nuts

```
ignore := Channels{{cnf.DestChan}}
```

But that's crazy :)